content/rc/administration/setup/create-database.md

```diff
@@ -14,40 +14,36 @@ Once you have a subscription, you can easily create a database in Redis Cloud by
 1. In the Redis Cloud menu, click **Databases**.
 1. In the subscription where you want to add the database, click ![Add](/images/rs/icon_add.png#no-click "Add").
 1. Configure the database:
-    1. Enter a **Database Name** that is up to 40 characters long.
-    1. Protocol - Select whether the database uses **Redis** or **Memcached**.
-    1. Enter a **Memory Limit**. Please take in mind that replication is
+    - **Name** - Enter a **Database Name** that is up to 40 characters long.
+    - **Protocol** - Select whether the database uses **Redis** or **Memcached**.
+    - **Memory** - Enter a **Memory Limit**. Please take in mind that replication is
         on.
-    1. If your database uses [Redis on
-        Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}),
-        enter the average item size(in bytes). This helps us optimaize your database.
-    1. In **Throughput by**, select the definition of thoughput as:
-    1. Ops/sec - Enter the required **Max Throughput** between 1000 and 10000000.
-    1. Shards - Enter the number of **Shards** you require for the database.
-    1. **Replication** is enabled by default so that each shard has a slave shard.
-        If you do not require slave shards, for example in a caching database,
-        disable replication.
-    1. In **Data Persistence**, select when the data is saved to [persistent storage]({{< relref "rc/concepts/data-persistence.md" >}}) :
+    - **Redis on Flash** - If your database uses [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}),
+        enter the average item size (in bytes). This helps us optimize your database.
+    - **Throughput** - In **Throughput by**, select the definition of throughput as:
+        - Ops/sec - Enter the required **Max Throughput** between 1000 and 10000000.
+        - Shards - Enter the number of **Shards** you require for the database.
+    - **Replication** - By default, each shard has a slave shard.
+        If you do not require slave shards, for example in a caching database, disable replication.
```

Comment:
If you do not require replica shards, then disable replication.

Reply:
same